gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -261,6 +261,74 @@ public boolean isOpen() {
         .getReadChannel(resourceId.getBucketName(), resourceId.getObjectName(), readOptions);
   }
 
+  @Override
+  public SeekableByteChannel open(
```

Comment:
Could you elaborate on this use-case for this method ? 

(especially `new SeekableByteChannel() ` seems sketchy )

Reply:
It is the overriden open method, when we use item info